config/storage.yml

```diff
@@ -9,10 +9,10 @@ local:
 # Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
 # amazon:
 #   service: S3
-#   access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
-#   secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
-#   region: us-east-1
-#   bucket: your_own_bucket
+#   access_key_id: ENV["AWS_ACCESS_KEY_ID"]
```

Comment:
@josephelias-rubyians  Are you storing the files in Active Storage  tables or in AWS S3?  I won't be able to give any AWS S3 credentials